client/src/components/LinkField/LinkField.js

```diff
@@ -190,7 +243,22 @@ const LinkField = ({
         types={types}
         canCreate={canCreate}
       /> }
-        <div> { renderLinks() } </div>
+    { isMulti && <div className={linksClassName}>
+      <DndContext modifiers={[restrictToVerticalAxis, restrictToParentElement]}
+        sensors={sensors}
+        collisionDetection={closestCenter}
+        onDragStart={handleDragStart}
+        onDragEnd={handleDragEnd}
+      >
+        <SortableContext 
+          items={linkIDs}
+          strategy={verticalListSortingStrategy}
+        >
+          {links}
+        </SortableContext>
+      </DndContext>
+    </div> }
+    { !isMulti && <div>{ renderLinks() }</div>}
```

Comment:
```suggestion
    { !isMulti && <div>{ links }</div>}
```
I presume this was the point of turning it into a variable